### PR TITLE
better textures

### PR DESCRIPTION
**Pull request by nurgan**

### CODE REVIEW

File: levels/grandmas-awakening.json [hunk 1]
```diff
@@ -253,6 +254,14 @@
...
          "name": "Cookie_Texture",
          "filename": "cookie_texture.jpg"
+      },
+      {
+         "name": "Ground",
+         "filename": "ground.jpg"
+      },
+      {
+         "name": "Ground_Hell",
+         "filename": "ground_hell.jpg"
```
Comment (rgarmsen2295) [inline levels/grandmas-awakening.json:269]: Friggin' awesome :)

**Review by rgarmsen2295 — CHANGES_REQUESTED** [review]
Love the new textures/skybox!  However, I seem to be defaulting to the hell texture plane on my machine, I assume that's not what you're seeing?

**Review by rgarmsen2295 — APPROVED** [review]

**Comment by rgarmsen2295** [comment]
Nvm, saw your comment in slack and ran it a few more times, seems fine.  Must have just hit the tree switch button on accident

File: levels/grandmas-awakening.json (hunk 1 above)
Comment (joncatanio) [reply levels/grandmas-awakening.json:269]: yes!!!

**Pull request merged**